libs/server.js

```diff
@@ -126,6 +125,7 @@ function startServer(){
 	naming.configure(miaou);
 
 	var cookieParser = require('cookie-parser')(miaou.config.secret);
+	var sessionStore = new RedisStore(miaou.config.redisStore);
```

Comment:
Some settings are more important than others :) Having a session secret is necessary. Having a special redis conf isn't. But let's leave the decision to @Canop